spec/controllers/auth/registrations_controller_spec.rb

```diff
@@ -44,27 +44,98 @@
     end
   end
 
-  describe 'GET #update' do
-    let(:user) { Fabricate(:user) }
+  describe 'PUT #update' do
+    let(:current_password) { Faker::Internet.password }
```

Comment:
I would just use explicit values throughout the file instead of using faker (and adding randomness to the spec).

Reply:
Randomness in tests is better in my book, but removing it is fine too :)